File: .py/Program/Petshop.py
# Class Petshop
class Petshop:
    # Atribut
    __ID = 0
    __nama = ""
    __kategori = ""
    __harga = 0

    # Method
    # Constructor
    def __init__(self, ID = 0, nama = "", kategori = "", harga = 0):
        self.__ID = ID
        self.__nama = nama
        self.__kategori = kategori
        self.__harga = harga

    # Setter and Getter
    def setID(self, ID):
        self.__ID = ID

    def getID(self):
        return self.__ID

    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama

    def setKategori(self, kategori):
        self.__kategori = kategori

    def getKategori(self):
        return self.__kategori

    def setHarga(self, harga):
        self.__harga = harga

    def getHarga(self):
        return self.__harga

File: .php/Program/Petshop.php
<?php

// Class Petshop
class Petshop {
    // Atribut
    public $ID = 0;
    public $nama = '';
    public $kategori = '';
    public $harga = '';
    public $foto = '';

    // Constructor
    public function __construct($ID = 0, $nama = '', $kategori = '', $harga = 0, $foto = '') {
        $this->ID = $ID;
        $this->nama = $nama;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->foto = $foto;
    }

    // SETTER & GETTER
    public function setID ($ID) { $this->ID = $ID; }
    public function getID () { return $this->ID; }

    public function setNama ($nama) { $this->nama = $nama; }
    public function getNama () { return $this->nama; }

    public function setKategori ($kategori) { $this->kategori = $kategori; }
    public function getKategori () { return $this->kategori; }

    public function setHarga ($harga) { $this->harga = $harga; }
    public function getHarga () { return $this->harga; }

    public function setFoto ($foto) { $this->foto = $foto; }
    public function getFoto () { return $this->foto; }
}

?>